### PR TITLE
Color, Accessibility, Typography, & Logo Guidelines

### PR DESCRIPTION
**Pull request by britanyponvelle**
Updated color guidelines and color palette asset with YAML and front matter

### CODE REVIEW

**Comment by Snugug** [comment]
@britanyponvelle See the inline comment; you've got an error!

**Comment by britanyponvelle** [comment]
oh hey that's so cool! I will fix that

**Comment by Snugug** [comment]
Okay, it's still failing, but it's a known build failure, so this is GTG from an integration perspective. Someone want to review for content?

**Comment by britanyponvelle** [comment]
@Snugug can you please explain what you mean? My brain did not register that sentence :(

**Comment by Snugug** [comment]
:tongue: Haha, yah, no worries!

So, see that red `x` next to your update and the `x Failed` in the yellow box? That tells me that our Continuous Integration system (CI) has failed, meaning there's an error somewhere. GitHub, with the yellow and the giant `Merge with caution!` message is telling me that while it can merge your changes in automatically, I may not want to because the system has failed. I went and looked at the results of the CI by clicking on the red `x` next to your update, and I can see the whole process that happened to get it to tell me that the build has failed. I can see that the error there is one that I know exists, specifically it's an issue with our Sass compiling that we need to wait for someone else outside our project to update before it gets fixed. This means that the failure isn't due to anything you've done, so I'm happy to merge it in. If you click the first `x` here, you'll see the difference: the first time there was an error early on from `yaml-js` on `line 46 column 5`, pointing me to your YAML as the source of the problem. You fixed that!

So, with all of that said, I think this pull request is Good to Go (GTG) from a technical standpoint. Just want someone to review the actual content to ensure that's GTG too. Once we get the :+1:s, we can merge

**Comment by britanyponvelle** [comment]
Ah, ok great! Thanks for explaining that for me. 

@channel: :+1: ?

File: language/color-guidelines.md [hunk 1]
```diff
@@ -1,32 +1,92 @@
...
+        personality: 
+          - Innovation
+          - Utility
+      - palette: 'Teal'
+        number: 60
+        personality:
+          - Awareness
+          - Approchability
+  accent-colors:
+    instructions: Choose a primary and secondary color
```
Comment (bobbinrobyn) [inline language/color-guidelines.md]: Choose an accent color. Add hues as necessary.

File: language/color-guidelines.md [hunk 2]
```diff
@@ -1,32 +1,92 @@
...
+      - palette: 'Orange'
+        number: 50
+        personality: 
+          - Innovation
+          - Utility
+      - palette: 'Teal'
+        number: 60
+        personality:
+          - Awareness
+          - Approchability
```
Comment (bobbinrobyn) [inline language/color-guidelines.md]: Approachability

File: language/visual-accessibility-guidelines.md [hunk 3]
```diff
@@ -1,6 +1,34 @@
...
+---
 
-**Resources:**  
-1. [Color Contrast Analyzer](http://www.paciellogroup.com/resources/contrastanalyser/)
-2. [Contrast Ratio Analyzer](http://leaverou.github.io/contrast-ratio/)
\ No newline at end of file
+# Accessibility
+When choosing colors and contrast ratios for typography it is important to adhere to the [IBM Design Language recommendations for accessibility](http://www.ibm.com/design/language/framework/visual/color.shtml#contrast), as we must meet [WCAG 2.0 AA requirements](http://www.w3.org/TR/WCAG20/). High color contrast helps users who are partially or completely color blind see differences between certain colors. It creates a strong visual hierarchy and improved usability for everyone. Never rely on color as the only visual means of conveying information, indicating an action, prompting a response, or distinguishing a visual element.  
+
+![Accessibile Log In Page](images/visual/color-applying-light.svg)
```
Comment (bobbinrobyn) [inline language/visual-accessibility-guidelines.md]: Accessible

**Comment by bobbinrobyn** [comment]
Okay, just those three changes from me! Someone else (looking at you @kathrynmcelroy) might want to look over it as well, but I feel good about merging it in.

File: language/color-guidelines.md [hunk 4]
```diff
@@ -1,32 +1,92 @@
...
+type: guideline
+title: Color
+resources:
+  files:
+    - name: IBM Color Palette ASE
+      source: /assets/visual/ibm-color-palette.ase
+variables:
+  palette-application:
+    light:
+      title: Light Palette
```
Comment (kathrynmcelroy) [inline language/color-guidelines.md]: Example: Light Palette

File: language/color-guidelines.md [hunk 5]
```diff
@@ -1,32 +1,92 @@
...
+      image: /images/visual/light-palette.svg
+      description: 'Primary: Blue 60, Secondary: Orange 50, Accent: Teal 40'
+      caption: 'Consider users who are sensitive to brightness and high contrast colors by using a neutral value as a background color instead of pure white.'
+    dark:
+      title: Dark Palette
```
Comment (kathrynmcelroy) [inline language/color-guidelines.md]: Example: Dark Palette

File: language/color-guidelines.md [hunk 6]
```diff
@@ -1,32 +1,92 @@
...
 ## Colors for Watson
-The colors chosen for IBM Watson are flexible and distinguish Watson as a brand across all user experiences. The colors chosen for IBM Watson were developed to support the Watson brand attributes,  reinforce Watson's personality (link to personality), and set the tone of your product. 
 
-**Visual Examples:**  
-![Brand Color Meanings](/images/visual/brand-color-meaning.png)
+The colors chosen for IBM Watson are flexible and distinguish Watson as a brand across all user experiences. The colors chosen for IBM Watson were developed using color psychology to support the Watson [brand attributes](brand-attributes.html), reinforce Watson's personality, and set the tone of each specific product.
 
-### Choosing A Color Palette
-Choose a primary color, a secondary color, and an accent color. Additional tints and shades of your palette can help to unify your design. These values can add variety to your palette without additional colors. Use the IBM Design Language is available for a wider variety of colors, values, and neutrals.
+{{> primary-color-display primary-colors}}
```
Comment (kathrynmcelroy) [inline language/color-guidelines.md]: Should we add the resource box after this?

File: language/typography-guidelines.md [hunk 7]
```diff
@@ -1,29 +1,52 @@
...
+## Helvetica Neue for IBM
+As a rule and wherever possible, use Helvetica Neue. It is the font of science and the information age, with a precision and objectivity that commands respect. We lean on Helvetica Neue to do the hard work of conveying information, specifications, and the basics. Its clean confidence makes it ideal for our product design.
 
-*We are currently in the process of inquiring about full usage rights and licensing for Helvetica Neue for IBM. During this time, we're unable to provide downloadable font assets. Please defer to the Cross-Platform Fontstack Options below for comparable fontstacks.*
+We are currently in the process of inquiring about full usage rights and licensing for Helvetica Neue for IBM. During this time, we're unable to provide downloadable font assets. Please defer to the Cross-Platform Fontstack Options below for comparable fontstacks.
```
Comment (kathrynmcelroy) [inline language/typography-guidelines.md]: How to we indicate that this is an aside?

**Comment by britanyponvelle** [comment]
@kathrynmcelroy I think we would add a resource box here. Like this?

```
{{> SOME-FUNCTION-HERE resources}}
```

**Comment by kathrynmcelroy** [comment]
that looks good! Finished commenting. Once edited, ready for :+1:

**Comment by britanyponvelle** [comment]
@bobbinrobyn @kathrynmcelroy See changes there ^^

**Comment by Snugug** [comment]
@britanyponvelle Updates for you

**Comment by britanyponvelle** [comment]
Ahh, ok! Thanks Sam. I will updated this tomorrow.

**Comment by Snugug** [comment]
As per #152, can you please move the markdown files to the following locations?

`color-guidelines.md` -> `guidelines/style/color`
`typography-guidelines.md` -> `guidelines/style/typography`
`visual-accessibility-guidelines.md` -> `guidelines/style/accessibility`
`wordmark-logotype-guidelines.md` -> `guidelines/branding/wordmark`

Thank you much!

**Comment by Snugug** [comment]
Also, can you please move your assets around?

Each asset should go in a sub-folder for the item it's representing, so colors should go in `images/colors`, wordmarks in `images/wordmarks`. etc… Same for things in `assets` and `videos`

**Comment by britanyponvelle** [comment]
@Snugug will all of the folders still be housed within the "language"?

**Comment by britanyponvelle** [comment]
One more question: right now my images are in `language/images/visual`. Should I delete the `visual` folder and add `color`, `typography`, etc. folders instead?

**Comment by Snugug** [comment]
Yup, everything in `language`. And yup to the second question too!

**Comment by britanyponvelle** [comment]
Awesome, thanks! Working on this now.

**Comment by Snugug** [comment]
Awesome!

**Comment by ryanbrownhill** [comment]
Resolves #154

**Pull request merged**